code/game/objects/items/robot/robot_parts.dm

```diff
@@ -252,7 +252,7 @@
 			if(!user.temporarilyRemoveItemFromInventory(W))
 				return
 
-			var/mob/living/silicon/robot/O = new /mob/living/silicon/robot/nocell(get_turf(loc))
+			var/mob/living/silicon/robot/O = new /mob/living/silicon/robot/nocell(get_turf(loc), FALSE)
```

Comment:
And the logic here?

Reply:
The idea is to connect to an AI only when intended so it doesn't connect to a different AI for a split second. I need to take a second look at the code that comes after it though.